src/pages/reader/epub.vue

```diff
@@ -248,7 +359,8 @@ export default {
       }
     },
     async searchEntireBook() {
-      if (this.searchText.length < 3) return [];
+      if (!this.searchText.length) return [];
```

Comment:
I think this magic number here is to avoid single character search hanging the whole page, maybe we need debounce and limit search result if we remove it

Reply:
suggest to keep the  `< 3` condition if we don't want to handle this for now
ideally we check `< 3` for alphanumeric char and skip the check for non ascii char